libcaf_net/caf/net/endpoint_manager_impl.hpp

```diff
@@ -48,10 +52,18 @@ class endpoint_manager_impl : public endpoint_manager {
 
   // -- properties -------------------------------------------------------------
 
+  application_type& application() {
+    return transport_.application();
+  }
+
   transport_type& transport() {
     return transport_;
   }
 
+  endpoint_manager_impl& manager() {
```

Comment:
What is this getter for?

Reply:
We need to have a reference/pointer to the manager for creating proxies.